src/DotNet.ReproducibleBuilds/DotNet.ReproducibleBuilds.targets

```diff
@@ -43,13 +34,13 @@
   </PropertyGroup>  
 
   <PropertyGroup>
-    <_ReproducibleBuildsMSBuildMinVersion>16.10.0</_ReproducibleBuildsMSBuildMinVersion>
+    <_ReproducibleBuildsMSBuildMinVersion>17.8.0</_ReproducibleBuildsMSBuildMinVersion>
+    <_ReproducibleBuildsMSBuildVersion>$(MSBuildVersion)</_ReproducibleBuildsMSBuildVersion><!-- Store in a non-reserved property for testing -->
```

Comment:
If anyone knows of a better way to test this without creating a new property, please let me know.